01-introduction.Rmd

```diff
@@ -1,6 +1,5 @@
 # Introduction {#intro}
 
-<!-- to do things sounds rather vague without context-->
 This book is about harnessing the power of modern computers to make the most of geographic data.
```

Comment:
Fair point but I think it may be better in some ways than the alternative you propose: `make the most of` is also vague and, to my ear, less exciting and fun than *do things*. Any other suggestions?

Thoughts on this @Nowosad?